presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -311,6 +312,11 @@ private boolean isTablesEnumeratingTable(SchemaTableName schemaTableName)
         return ImmutableSet.of(TABLE_COLUMNS, TABLE_VIEWS, TABLE_TABLES, TABLE_TABLE_PRIVILEGES).contains(schemaTableName);
     }
 
+    private boolean isColumnsEnumeratingTable(SchemaTableName schemaTableName)
```

Comment:
I think it should be called `isTablesEnumeratingTable` as it enumerates tables to get the list of columns or privileges. Then current `isTablesEnumeratingTable` should be renamed to `isSchemasEnumeratingTable` because it enumerates schemas to get lists of tables or views and `isSchemasEnumeratingTable` should not contain `columns` and `table_privileges` tables..